script.js

```diff
@@ -11,4 +11,137 @@ document.querySelectorAll(".nav-link").forEach(n => n.
+    liveDemo: "#",
+    sourceCode: "",
+    title1: "Facebook 360",
+    title2: "Facebook 360",
+},
+{
+    description: "A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.",
+    description2: "A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.",
+    img1: "Snapshoot Portfolio (1).png",
+    img2: "Snapshoot Portfolio (1).png",
+    technologies: ["Html", "Css", "Javascript"],
+    liveDemo: "#",
+    sourceCode: "",
+    title1: "Uber Navigation",
+    title2: "Uber Navigation",
+}
+]
+
+const section = document.querySelector(".works")
+const popup = document.querySelector(".popup-window")
+
+project.forEach((projects) => {
+    const article = document.createElement('article')
+    const divImg = document.createElement('div')
+    const img = document.createElement('img')
+    const elements = document.createElement('div')
+    const button = document.createElement("button")
+    img.src = `${projects.img1}`
+    article.classList.add('card1')
+    button.classList.add('btn')
+    elements.classList.add('elements1')
+    button.innerText = 'See Project'
+
+    elements.innerHTML += `
+    <h2 class="title">${projects.title1}</h2>
+                        <ul class="highlights">
+                            <li>CANOPY</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>Back End Dev</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>2015</li>
+                        </ul>
+                        <p class="description">${projects.description}</p>
+                        <ul class="languages">
+                            <li class="stack">${projects.technologies[0]}</li>
+                            <li class="stack">${projects.technologies[1]}</li>
+                            <li class="stack">${projects.technologies[2]}</li>
+                        </ul>
+    `
+    elements.append(button)
+    divImg.append(img)
+    article.append(divImg, elements)
+    section.append(article)
+
+    button.addEventListener("click", () => {
+        popup.style.display = 'flex'
+        section.style.filter = 'blur(5px)'
+        document.body.style.overflow = 'hidden'
+        popup.innerHTML = `
+    <div class="popup-header">
+                    <h2 class="popup-title">${projects.title1}</h2>
+                    <div class="cancel-icon"><img src="./ICONS/cancel-icon.png"></div>
+                </div>
+                <ul class="popup-highlights">
+                    <li>CANOPY</li>
+                    <li><img class="counter" src="Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img class="counter" src="Counter.png"></li>
+                    <li>2015</li>
+                </ul>
```

Comment:
- [ ] Please adjust the spacing, color, and font size of this section **on the mobile view** to better match the Figma template.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/195358733-6b8774ff-714b-4e9d-b130-2270b901b6d8.png) | ![image](https://user-images.githubusercontent.com/62290545/195358827-e2a6e68a-27eb-413b-9c80-4df88e7361c5.png) |